service/src/main/resources/changelog/changesets/20230105_run_set_prepopulation_target_workflow_4.yaml

```diff
@@ -0,0 +1,227 @@
+databaseChangeLog:
+  - changeSet:
+      id: "run_set_prepopulation_target_workflow_4"
+      author: kpierre
+      changes:
+        - insert:
+            tableName: method
+            columns:
+              - column:
+                  name: method_id
+                  value: 00000000-0000-0000-0000-000000000004
+              - column:
+                  name: name
+                  value: Target Workflow 4
+              - column:
+                  name: description
+                  value: Target Workflow 4
+              - column:
+                  name: created
+                  valueDate: "CURRENT_TIMESTAMP()"
+              - column:
+                  name: method_source
+                  value: Github
+              - column:
+                  name: last_run_set_id
+                  value: null
+        - insert:
+            tableName: method_version
+            columns:
+              - column:
+                  name: method_version_id
+                  value: 40000000-0000-0000-0000-000000000004
```

Comment:
is this a convention we're using for the version? just curious where this version ID comes from